setup.py

```diff
@@ -0,0 +1,30 @@
+from os import path
+
+from setuptools import find_packages, setup
+
+pkg_name = next(p for p in find_packages() if "." not in p)
+here = path.abspath(path.dirname(__file__))
+
+with open(path.join(here, "README.md"), "r") as f:
+    long_description = f.read()
+
+with open(path.join(here, "requirements.txt")) as f:
+    requirements = f.read().splitlines()
+
+with open(path.join(here, pkg_name, "version.py")) as f:
+    exec(f.read())
+
+setup(
+    name=pkg_name.replace("_", "-"),
+    version=__version__,  # noqa: F821
+    description="DataJoint Elements for Lab Management",
```

Comment:
This should be either `DataJoint Elements for Cell Matching` or `DataJoint Element for Volume Registration`.